docs/reference/toc.yml

```diff
@@ -9,8 +9,15 @@ toc:
       - file: solutions-ecommerce-search-page.md
   - file: tutorials.md
     children:
-      - file: tutorials-app-search.md
       - file: tutorials-elasticsearch.md
+        children:
+          - file: tutorials-elasticsearch-setup-cloud.md
+          - file: tutorials-elasticsearch-setup-index.md
+          - file: tutorials-elasticsearch-install-connector.md
+          - file: tutorials-elasticsearch-configure-search-ui.md
+          - file: tutorials-elasticsearch-production-usage.md
+          - file: tutorials-elasticsearch-customise-query.md
```

Comment:
not sure if wanted to swap these two in toc?

Reply:
No, thank you, it was intentional, because previous tutorials are related and last one tutorials more like a standalone